onnxruntime/core/providers/openvino/qdq_transformations/qdq_stripping.h

```diff
@@ -17,7 +17,8 @@ Status CreateModelWithStrippedQDQNodes(const GraphViewer& src_graph,
                                        const logging::Logger& logger,
                                        bool enable_ovep_weight_sharing,
                                        /*out*/ std::unique_ptr<onnxruntime::Model>& model,
-                                       /*out*/ sw& shared_weights);
+                                       /*out*/ sw& shared_weights,
+                                       bool enable_ovep_qdq_optimizer);
```

Comment:
Shouldn't enable_ovep_qdq_optimizer be placed before all output parameters?

Reply:
Noted. Will take it up in the next PR.